docs/content/documentation/systems/Kernels/framework/MaterialDerivativeTestKernel.md

```diff
@@ -1,7 +1,49 @@
 <!-- MOOSE Documentation Stub: Remove this when content is added. -->
 
 # MaterialDerivativeTestKernel
-!syntax description /Kernels/MaterialDerivativeTestKernel
+
+## Description
+
+`MaterialDerivativeTestKernel` is mostly useful as a demonstration object. Its
```

Comment:
No, it is a generic test Kernel, that allows checking the correctness of the derivative material properties using the built in Jacobian checks in the test harness!